src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -89,6 +80,130 @@ public void setUp() throws OpenTokException {
         sdk = new OpenTok.Builder(apiKey, apiSecret).apiUrl(apiUrl).build();
     }
 
+    @Test
+    public void testSignalAllConnections() throws OpenTokException {
+        String sessionId = "SESSIONID";
+        String path = "/v2/project/" + apiKey + "/session/" + sessionId + "/signal";
+        stubFor(post(urlEqualTo(path))
+                .willReturn(aResponse()
+                        .withStatus(204)));
+
+        SignalProperties properties = new SignalProperties.Builder().type("test").data("Signal test string").build();
+        sdk.signal(sessionId, properties);
```

Comment:
Since the `signal` method throws an exception, we should implement a try/catch statement when sending the signal.